en/upgrades/decimal-upgrade-to-2-03.rst

```diff
@@ -1,4 +1,6 @@
 Decimal Upgrade to 2.03
 =======================
 
+The next decimal upgrade of the IATI Standard (to version 2.03) will take place from Friday 12 May through to early October 2017. The full timetable below outlines each phase of the upgrade process and  includes information on when input is invited from the IATI Community. We anticipate the end of this process and the release of version 2.03 on **Tuesday 3 October 2017**.
```

Comment:
Thanks @amy-silcock - yep sounds good but needs to say version 2.03 in brackets, and let's stick with not using th and rd on the dates as that seems to be the fashion/is how we have it in the table!